api/src/main/java/de/valtech/aecu/api/groovy/console/bindings/filters/FilterBy.java

```diff
@@ -21,12 +21,14 @@
 import javax.annotation.Nonnull;
 
 import org.apache.sling.api.resource.Resource;
+import org.osgi.annotation.versioning.ConsumerType;
 
 /**
  * Interface for AECU binding filters.
  * 
  * @author Roxana Muresan
  */
+@ConsumerType
```

Comment:
Muss das ConsumerType sein?

Reply:
I think yes because we have "filterWith(FilterBy filter)" in content migration API. So users can implement their own filters.